main.tf

```diff
@@ -1,18 +1,46 @@
 module "tags" {
   source = "git::https://github.com/rhythmictech/terraform-terraform-tags.git?ref=v0.0.2"
   tags   = var.tags
-
```

Comment:
want space before multiline arg

Reply:
Yeah, I caught this in the code review on Roberto's PR. I'll merge his changes into my branch before I finalize this